h/streamer/tweens.py

```diff
@@ -0,0 +1,19 @@
+__all__ = ["close_db_session_tween_factory"]
+
+
+def close_db_session_tween_factory(handler, registry):
+    """Return the close_db_session_tween."""
+
+    def close_db_session_tween(request):
+        """Close the sqlalchemy session.
+
+        This tween runs late in the websocket app's Pyramid request processing
+        cycle and makes sure that any DB connections that were opened during
+        the request get closed.
+        """
+        try:
+            return handler(request)
+        finally:
+            request.db.close()
```

Comment:
If `handler()` raises we don't catch it (there is no `except` here) but this line still gets run

Reply:
Is it worth mentioning this is only here because of exclog?